zkvm/spec/ZkVM.md

```diff
@@ -1867,6 +1867,14 @@ Payment channel overview:
 
 TBD: specifics.
 
+Payment channel is specified as follows:
```

Comment:
Could you remove this duplicate chunk? It seems, it accidentally came from a manual merge conflict resolution.